client/me/account/main.jsx

```diff
@@ -957,6 +959,23 @@ class Account extends Component {
 				<SectionHeader label={ translate( 'Interface settings' ) } />
 				<Card className="account__settings">
 					<form onSubmit={ this.saveInterfaceSettings }>
+						{ config.isEnabled( 'dashboard/v2' ) && (
+							<FormFieldset className="account__settings-admin-home">
+								<FormLabel id="account__new-hosting-dashboard">
```

Comment:
Looks like the IDs in these form fields are to allow easy hyperlinking from other pages.